spec/system/admin/order_cycles/simple_spec.rb

```diff
@@ -258,6 +274,18 @@
         click_button 'Add supplier'
         expect(page).to have_content "Permitted supplier"
 
+        within("tr.supplier-#{supplier_permitted.id}") { click_button 'Add fee' }
+        expect(page).to have_select(
+          "order_cycle_incoming_exchange_1_enterprise_fees_0_enterprise_id", minimum: 1
+        )
+        page.find("select#order_cycle_incoming_exchange_1_enterprise_fees_0_enterprise_id")
+          .all('option[label]').first.select_option
+        sleep 1
```

Comment:
Is this to wait for the next select to load? Usually have_* matchers will wait for the condition to pass, so I thought a sleep wouldn't be necessary.
But.. strangely this spec doesn't work on my computer. Do you  have any ideas why?

`expected to find visible select box "order_cycle_incoming_exchange_1_enterprise_fees_0_enterprise_fee_id" that is not disabled with options ["", "Supplier distributor fee1", "Supplier distributor fee3"] but there were no matches.`

`Also found "Supplier distributor fee1\nSupplier distributor fee3\nSupplier distributor fee4\nSupplier distributor fee2", which matched the selector but not all filters.`

`Expected options ["", "Supplier distributor fee1", "Supplier distributor fee3"]
found ["", "Supplier distributor fee1", "Supplier distributor fee3", "Supplier distributor fee4", "Supplier distributor fee2"]`
![Screenshot 2024-07-10 at 9 42 26 am](https://github.com/openfoodfoundation/openfoodnetwork/assets/4188088/55b3416c-6863-41d1-bf2e-ce4543915936)

Reply:
Weird. From your screenshot it means the implementation isnt working. It works on my other two machines (my current windows has an older wsl which doesnt have X/wayland)